lib/screens/navigator/slide_left_route.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class SlideLeftRoute extends PageRouteBuilder {
```

Comment:
czy na pewno musisz definiować własną animację między ekranami? te standardowe zazwyczaj działają świetnie plus obsługują iOSową nawigację

Reply:
nie musiałem, chciałem zobaczyć jak to działa i się tym pobawić